### PR TITLE
Release IIB v6.7.1

### PR DESCRIPTION
**Pull request by yashvardhannanavati**
Refers to CLOUDDST-16213

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 6.7.1
```
Comment (chandwanitulsi) [inline CHANGELOG.md:9]: Maybe 6.8 would be better?
The release started a bug fix but more got added

Comment (yashvardhannanavati) [reply CHANGELOG.md:9]: But it's still bug fixes and no new features are being added. So I thought it might be better to keep it 6.7.1?

Comment (JAVGan) [reply CHANGELOG.md:9]: Aside from `opm` upgrade everything is just a bugfix, so `6.7.1` makes sense IMO as we are mostly adding fixes...

I'm not sure if just a new `opm` binary counts as a "feature added" as we just upgraded a dependent component, but I'm fine with 6.8 if you believe this is a "feature" change

Comment (JAVGan) [reply CHANGELOG.md:9]: But honestly... I'm more up to `6.7.1` than `6.8` due to most of it being fixes

**Review by JAVGan — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (chandwanitulsi) [reply CHANGELOG.md:9]: yeah OPM update was making me inclined towards 6.8. I don't have strong objections to 6.7.1 as well. Happy to go with majority

**Review by chandwanitulsi — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Fix dev env compose files
+- Fix bundle_replacements bug in regenerate-bundle request
+- Upgrade OPM to 1.26.2 in dev env
+- Fix buildah bud command retries
```
Comment (chandwanitulsi) [inline CHANGELOG.md:13]: I just realized we missed adding "Adding retry and port check when opm serve/opm registry serve is called" in changelog

**Pull request merged**